projects/dynamicforms/src/core/resolvers/user-address-prefill-resolver.ts

```diff
@@ -0,0 +1,28 @@
+import { UserAddressService } from '@spartacus/core';
+import { map } from 'rxjs/operators';
+import { Injectable } from '@angular/core';
+import { PrefillResolver } from './prefill-resolver.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserAddressPrefillResolver implements PrefillResolver {
+  constructor(protected userAddressService: UserAddressService) {}
+
+  getPrefillValue(fieldPath: string) {
+    const attributes = fieldPath.split('.');
+    let currentValue;
+    return this.userAddressService.getAddresses().pipe(
+      map(address => {
+        currentValue = address[0];
+        if (!currentValue) {
+          return ' ';
+        }
+        attributes.forEach(attribute => {
+          currentValue = currentValue[attribute];
```

Comment:
Maybe try to set it like `currentValue = currentValue[attribute] || '  ';` and avoid that if above?

Reply:
Can't be like that because error is thrown when you try to fetch attribute form "undefined" object

Reply:
I see, what about `currentValue = address[0] || '  ' ` on line 17? Would that work?

Reply:
It would be the same, in that case it will try to fetch attribute from empty string and error will occur again.